fpm/src/fpm.F90

```diff
@@ -3,10 +3,31 @@ module fpm
 private
 public :: print_help, cmd_build
 
+integer, parameter :: os_linux = 1
+integer, parameter :: os_macos = 2
+integer, parameter :: os_windows = 3
```

Comment:
Style suggestion: Do you like all caps for constants like in Python, e.g. `OS_LINUX`?

Reply:
Doesn't matter to me, I think i've been using lowercase, but upper case would also work.

Reply:
I use uppercase now.